### PR TITLE
Creating different card types for outgoing-webhook

### PR DESCRIPTION
**Pull request by AbhijeetThakur-msft**

### CODE REVIEW

File: samples/outgoing-webhook/nodejs/app.js [hunk 1]
```diff
@@ -3,46 +3,280 @@
 // Licensed under the MIT license.
 //
 const crypto = require('crypto');
-const sharedSecret = "<Security token generated by Microsoft Teams>"; // e.g. "+ZaRRMC8+mpnfGaGsBOmkIFt98bttL5YQRq3p2tXgcE="
+const sharedSecret = ""; // e.g. "+ZaRRMC8+mpnfGaGsBOmkIFt98bttL5YQRq3p2tXgcE="
```
Comment (v-chetsh) [inline samples/outgoing-webhook/nodejs/app.js:6]: Put some comment like "Security token generated by Microsoft Teams"

File: samples/outgoing-webhook/nodejs/app.js [hunk 2]
```diff
@@ -3,46 +3,280 @@
...
+    // Respond to the request
+    request.on('end', function() {
+        try {
+            // Retrieve authorization HMAC information
+            var auth = this.headers['authorization'];
+            // Calculate HMAC on the message we've received using the shared secret			
+            var msgBuf = Buffer.from(payload, 'utf8');
+            var msgHash = "HMAC " + crypto.createHmac('sha256', bufSecret).update(msgBuf).digest("base64");
+            // console.log("Computed HMAC: " + msgHash);
+            // console.log("Received HMAC: " + auth);
```
Comment (v-chetsh) [inline samples/outgoing-webhook/nodejs/app.js:27]: Remove console.log commented code.

File: samples/outgoing-webhook/nodejs/app.js [hunk 3]
```diff
@@ -3,46 +3,280 @@
...
+
+            response.writeHead(200);
+            if (msgHash === auth) {
+                var receivedMsg = JSON.parse(payload);
+
+				// The text received to webhook
+				var receivedText = receivedMsg.text;
+
+				// The message sent by webhook
+                var responseMsg = '';
```
Comment (v-chetsh) [inline samples/outgoing-webhook/nodejs/app.js:37]: Alignment issue

File: samples/outgoing-webhook/nodejs/app.js [hunk 4]
```diff
@@ -3,46 +3,280 @@
...
+
+                                }
+                            ]
+                        });
+                        break;
+
+					case (receivedText.indexOf("hero-card") != -1):
+                        // Creating hero card response
+                        responseMsg = JSON.stringify({
+                            "type": "message",
```
Comment (v-chetsh) [inline samples/outgoing-webhook/nodejs/app.js:82]: Align the content properly

File: samples/outgoing-webhook/nodejs/app.js [hunk 5]
```diff
@@ -3,46 +3,280 @@
...
+
+                                            }
+                                        ]
+                                    }
+                                }
+                            ]
+                        });
+                        break;
+
+					case (receivedText.indexOf("o365-card") != -1):
```
Comment (v-chetsh) [inline samples/outgoing-webhook/nodejs/app.js:185]: Switch  ... Case aligment is not proper

**Review by v-chetsh — APPROVED** [review]
Looks good

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**